### DISCUSSION TITLE
MultiSelect component as in shadcn website

### DISCUSSION THREAD

**Discussion by orelHAnyvision** [shadcn-ui/ui#3851] [General]
I saw this multiselect component in the examples page:
https://ui.shadcn.com/examples/tasks
you can see it in the 'status' and 'priority' filters

can someone post the code for this? couldn't find it

thanks!

**Comment by Steveb599** [comment]
you have it here
https://github.com/shadcn-ui/ui/tree/main/apps/www/app/(app)/examples/tasks/components
